src/components/custom/signup.js

```diff
@@ -50,55 +50,45 @@ export const Appearances = {
     public: 'public',
     newSignup: 'newSignup',
 }
-class Signup extends Component {
-    state = {
-        email: '',
-        is_submitting: false,
-        email_error_msg: '',
-        submit_status: '',
-        submit_error_msg: '',
-    }
-
-    validateEmail = (email) => {
+const Signup = (props) => {
```

Comment:
Since I'll be extending this later on 😆  Will it be possible if we can deconstruct the props here. Would be more clearer since we can see all the expected props to pass in